packages/cli/bigtest.json

```diff
@@ -1,7 +1,7 @@
 {
   "app": {
-    "command": "bigtest-todomvc 36000",
-    "port": 36000
+    "command": "node ../todomvc/dist/start.js 36000",
```

Comment:
`bigtest-todomvc` was not available on my path, causing the cli tests to fail.

Is there some additional environment set-up I'd need to do to allow this command to work?

Reply:
Did you run `yarn prepack` in the root of the repository?